packages/utils/src/structs.ts

```diff
@@ -0,0 +1,18 @@
+import { refine, Struct } from 'superstruct';
+
+/**
+ * Validate that a value has a minimum length.
+ *
+ * @param struct - The struct to validate.
+ * @param min - The minimum length.
+ * @returns The struct.
+ */
+export function minSize(struct: Struct<string>, min: number) {
```

Comment:
Already exists in superstruct https://docs.superstructjs.org/api-reference/refinements#size

Reply:
`size(string(), 1)` is a string with a size of exactly 1, not a minimum size unfortunately.

Reply:
Would `size(string(), 1, Infinity)` work?

Reply:
Ah, I missed that bit. Fixed!